freqtrade/wallets.py

```diff
@@ -61,7 +61,8 @@ def get_total(self, currency) -> float:
 
     def update(self) -> None:
 
-        balances = self._exchange.get_balances()
+        exchange: Exchange = get_exchange(self._config, self._exchange_name)
```

Comment:
This should be in the constructor, not done each time (we call this many times!)

before changing, read the main comment too though.